tests/test_odl.py

```diff
@@ -1304,7 +1304,9 @@ def do_get(url, headers):
         # it extracts format information from 'odl:license' tags and creates
         # LicensePoolDeliveryMechanisms.
 
+
         # The importer created 6 editions, pools, and works.
+        eq_({}, failures)
```

Comment:
Without this line, if there's a problem in the import code, the test will fail because fewer than 6 editions were imported. It'll be impossible to see what the actual problem was. If the failure dictionary is non-empty, then the actual failure messages will be output when nosetests explains why the test failed.